File: CodeForces/Java/785A. Anton and Polyhedrons.java
import java.util.Scanner;

/**
 *
 * @author Aniso
 */
public class Mavenproject1 {

    public static void main(String[] args) {

        Scanner in = new Scanner(System.in);
        int n = in.nextInt();
        long r = 0;
        for(int i=0;i<n;i++) {
           String s= in.next();
            if (s.equals("Tetrahedron")) {
                r += 4;
            } else if (s.equals("Cube") ) {
                r += 6;
            } else if (s.equals("Octahedron")) {
                r += 8;
            } else if (s.equals("Dodecahedron")) {
                r += 12;
            } else if (s.equals("Icosahedron") ) {
                r += 20;
            }
        }
        System.out.println(r);
    }
}

File: CodeForces/C++/785A. Anton and Polyhedrons.cpp
#include <iostream>

using namespace std;

int main()
{
    int n;
    cin>>n;
    long long r=0;
    string s;
    while(n--)
    {
        cin>>s;
        if(s=="Tetrahedron")
            r+=4;
        else if(s=="Cube")
            r+=6;
        else if(s=="Octahedron")
            r+=8;
        else if(s=="Dodecahedron")
            r+=12;
        else if(s=="Icosahedron")
            r+=20;
    }
    cout<<r;
    return 0;
}



